commit a0e4aec45458e1922263c992a8975fabe6100140
Author: Didier Raboud <odyx@debian.org>
Date:   Mon Jan 7 09:55:06 2013 +0100

    Ship cups-files.conf's manpage in cups
    
    - Update the configuration files split patch to also build the manpage;
    - Install the english manpage.
    
    Closes: #697543

diff --git a/debian/cups.install b/debian/cups.install
index 1daa312..7a7e735 100644
--- a/debian/cups.install
+++ b/debian/cups.install
@@ -28,6 +28,7 @@ usr/share/man/man5/classes.conf.5.gz
 usr/share/man/*/man5/classes.conf.5.gz
 usr/share/man/man5/cupsd.conf.5.gz
 usr/share/man/*/man5/cupsd.conf.5.gz
+usr/share/man/man5/cups-files.conf.5.gz
 usr/share/man/man5/cups-snmp.conf.5.gz
 usr/share/man/*/man5/cups-snmp.conf.5.gz
 usr/share/man/man5/mailto.conf.5.gz
diff --git a/debian/patches/split-configuration-files-STR4223.dpatch b/debian/patches/split-configuration-files-STR4223.dpatch
index 1bfecb0..93d6504 100755
--- a/debian/patches/split-configuration-files-STR4223.dpatch
+++ b/debian/patches/split-configuration-files-STR4223.dpatch
@@ -1467,16 +1467,30 @@ diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch'
  msgid "Folio"
  msgstr "對開紙"
 diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' cups~/man/Makefile cups/man/Makefile
---- cups~/man/Makefile	2012-12-29 12:25:12.000000000 +0100
-+++ cups/man/Makefile	2012-12-29 12:25:13.000000000 +0100
-@@ -39,6 +39,7 @@
+--- cups~/man/Makefile	2013-01-07 14:24:01.000000000 +0100
++++ cups/man/Makefile	2013-01-07 14:24:46.000000000 +0100
+@@ -39,8 +39,10 @@
  		ppdpo.$(MAN1EXT)
  MAN5	=	classes.conf.$(MAN5EXT) \
  		client.conf.$(MAN5EXT) \
 +		cups-files.conf.$(MAN5EXT) \
  		cups-snmp.conf.$(MAN5EXT) \
  		cupsd.conf.$(MAN5EXT) \
++		cups-files.conf.$(MAN5EXT) \
+ 		mailto.conf.$(MAN5EXT) \
+ 		mime.convs.$(MAN5EXT) \
+ 		mime.types.$(MAN5EXT) \
+diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' cups~/man/Makefile.l10n cups/man/Makefile.l10n
+--- cups~/man/Makefile.l10n	2013-01-07 14:24:00.000000000 +0100
++++ cups/man/Makefile.l10n	2013-01-07 14:24:31.000000000 +0100
+@@ -35,6 +35,7 @@
+ 		client.conf.$(MAN5EXT) \
+ 		cups-snmp.conf.$(MAN5EXT) \
+ 		cupsd.conf.$(MAN5EXT) \
++		cups-files.conf.$(MAN5EXT) \
  		mailto.conf.$(MAN5EXT) \
+ 		mime.convs.$(MAN5EXT) \
+ 		mime.types.$(MAN5EXT) \
 diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' cups~/man/cups-files.conf.man.in cups/man/cups-files.conf.man.in
 --- cups~/man/cups-files.conf.man.in	1970-01-01 01:00:00.000000000 +0100
 +++ cups/man/cups-files.conf.man.in	2012-12-29 12:25:13.000000000 +0100
